pkg/systemd/overview-cards/insights-poll-hack.py

```diff
@@ -0,0 +1,37 @@
+# Poll until /var/lib/insights/insights-details.json is 5 minutes
+# older than /etc/insights-client/.lastupload, then exit.
+
+# Calling "insights-details --check-results" only returns results
+# corresponding to the most recent upload some time after the upload,
+# but we don't know when exactly.  We assume that it will not take
+# more than 5 minutes.  However, we also want the results as soon as
+# they are available so we poll a couple of time before the 5 minutes
+# are up.
+
+# We poll fast for the first minute, and then slow down.  Also, there
+# is a absolute limit on how often we poll, in case something is wrong
+# with the time stamps (such as .lastupload being from 5 years in
+# future).
+
+import os
+import sys
+import subprocess
+import time
+
+def details_out_of_date():
+    if not os.access("/etc/insights-client/.lastupload", os.F_OK):
```

Comment:
FTR, it seems the insights devs want to fix this on their side, so we don't need that hack at all? (and otherwise, rewrite in plain shell)

Reply:
Yes, but let's see who is faster. :-)